integrations/cavern-adapter/src/money_market.rs

```diff
@@ -230,20 +230,11 @@ impl MoneyMarketCommand for Cavern {
 
     fn price(
         &self,
-        deps: Deps,
-        base: AssetInfo,
-        quote: AssetInfo,
+        _deps: Deps,
+        _base: AssetInfo,
+        _quote: AssetInfo,
     ) -> Result<Decimal, MoneyMarketError> {
-        let oracle_contract = &self.oracle_contract.clone().unwrap();
-        let price: moneymarket::oracle::PriceResponse = deps.querier.query_wasm_smart(
-            oracle_contract,
-            &moneymarket::oracle::QueryMsg::Price {
-                base: base.to_string(),
-                quote: quote.to_string(),
-            },
-        )?;
-
-        Ok(price.rate.try_into()?)
+        unimplemented!("Price query not implemented for Cavern, because cavern doesn't handle collaterals and doesn't have denoms inscribed normally")
```

Comment:
This should probably be an err instead of a panic?

Reply:
Yes